app/src/main/java/com/allan/boardbuddies/fragments/TextDialogFragment.java

```diff
@@ -0,0 +1,72 @@
+package com.allan.boardbuddies.fragments;
+
+import android.app.Dialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.graphics.Color;
+import android.graphics.drawable.ColorDrawable;
+import android.os.Bundle;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+import android.view.WindowManager;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.LinearLayout;
+
+import androidx.appcompat.app.AlertDialog;
+import androidx.fragment.app.DialogFragment;
+
+import com.allan.boardbuddies.R;
+
+public class TextDialogFragment extends DialogFragment {
+
+    private EditText editText;
+
+    public static TextDialogFragment newInstance(String in) {
+        TextDialogFragment f = new TextDialogFragment();
+
+        Bundle args = new Bundle();
+        args.putString("text", in);
```

Comment:
this was just placeholder for testing right? can remove?

Reply:
will need for editing function later. I will change to give empty value?